packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -129,11 +138,13 @@ export class GCFBootstrapper {
     const config = JSON.parse(payload);
     if (logging) {
       logger.info('custom logging instance enabled');
-      const LoggingOctokit = Octokit.plugin(LoggingOctokitPlugin);
+      const LoggingOctokit = Octokit.plugin(LoggingOctokitPlugin).plugin(
```

Comment:
this was required to continue giving us support for `context.github.config`.

Reply:
Instead of using `Octokit` from `@octokit/rest`, you can `import { ProbotOctokit } from 'probot'`, it will come with all the plugins and defaults that Probot is using internally

Reply:
You might also run into compatibility issues. Because Probot v10 is compatible with `@octokit/rest` v17, while you install v18 in your package.json. I would recommend to get rid of the `@octokit/rest` dependency altogether